### PR TITLE
New signIn design

### PR DESCRIPTION
**Pull request by ShikhaMishra9**
Resolve [AB#90847](https://dev.azure.com/EquinorASA/d02c350d-092d-41bf-a812-f05d5d8ad1b0/_workitems/edit/90847)

### CODE REVIEW

File: locales/english.json [hunk 1]
```diff
@@ -6,13 +6,15 @@
   "auth.prompt": "Please sign in",
   "auth.signin.title": "Sign in to {linebreak} Data Marketplace",
   "auth.signin.document.title": "Sign in to Data Marketplace",
-  "auth.signin.intro": "In order to use Data Marketplace, you need to sign in with your Equinor account.",
-  "auth.signin.first.time.visiting": "First time visiting? <a>Create an account by visiting the Collibra front page.</a> You can close the tab once you’ve signed in and the dashboard has been loaded.",
+  "auth.signin.first.time.visiting1": "First time visiting?",
+  "auth.signin.first.time.visiting2": "<a>Activate your account in collibra</a>",
```
Comment (aronhoyer) [inline locales/english.json:10]: isn't the anchor tag a little superfluous here?

**Review by aronhoyer — CHANGES_REQUESTED** [review]
dialog is too narrow on my browser (Safari 16.1):

Screenshot 2022-12-02 at 10 56 20

**Comment by aronhoyer** [comment]
additionally, i'm not taken to collibra when i click "Take me to Collibra" in the dialog. the cta is a button, not an anchor

**Comment by wenche** [comment]
> additionally, i'm not taken to collibra when i click "Take me to Collibra" in the dialog. the cta is a button, not an anchor

Lol, just took for granted that this was done :D

**Review by aronhoyer — APPROVED** [review]

**Pull request merged**